packages/xo-web/src/common/link.js

```diff
@@ -50,7 +50,7 @@ export class BlockLink extends Component {
     // FIXME: when https://github.com/facebook/react/issues/8529 is fixed,
     // remove and use onAuxClickCapture.
     // In Chrome ^55, middle-clicking triggers auxclick event instead of click
-    if (ref !== null) {
+    if (ref !== null && !!window.chrome && !!window.chrome.webstore) {
```

Comment:
I understand the PR is not ready yet, but if you keep these kind of changes, make sure to add (or update) a comment explaining why this is necessary.

Reply:
I added a comment, and I use this condition to use `auxclick` event in Chrome only.
`auxclick` is only compatible with Chrome ^55 and FireFox ^53.
In other browsers,  we use `onClickCapture` props.

Reply:
IMO, the workaround should have included this test from the beginning, it's still a Chrome related issue so the comment shouldn't mention Firefox which has always behaved the same way. (FF's behavior didn't change in v53, right?) If we want to keep track of this change, we can add a comment like "Other browsers may trigger both events" so we understand why we want to apply the workaround only in Chrome.